ATTRIBUTIONS/wasabi-api-attribution.xml

```diff
@@ -1282,7 +1282,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/api/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```

Comment:
nope. we have a post package/build script to sed out this string:

  see [ws]/bin/ATTRIBUTIONS_patch.sh

note: looking into using the mojo:license plugin.

thoughts?